src/deployment/deploy.py

```diff
@@ -1238,7 +1281,14 @@ def main() -> None:
         )
         states = rbac_only_states
     else:
-        states = full_deployment_states
+        if args.dotnet_deploy:
+            logger.info(
+                "deploying dotnet and python services for Azure functions"
+            )
+            states = full_deployment_states + [("dotnet-api", Client.deploy_dotnet_app)]
```

Comment:
careful here you are adding dotnet at the end of the list. it should be right after the python deployment. Because we have some steps taht occurs after we push the function

Reply:
good call, I'll revise this